Plugin/src/main/java/com/gmail/filoghost/holographicdisplays/bridge/protocollib/current/ProtocolLibHookImpl.java

```diff
@@ -84,7 +84,7 @@ public void onPacketSending(PacketEvent event) {
 					PacketContainer packet = event.getPacket();
 					Player player = event.getPlayer();
 					
-					if (player instanceof Factory) {
+					if (player instanceof ByteBuddyGenerated) {

```

Comment:
IMO it's not great to depend on an implementation detail. Would probably be better to use [event#isPlayerTemporary](https://github.com/dmulloy2/ProtocolLib/blob/master/src/main/java/com/comphenix/protocol/events/PacketEvent.java#L359)

Reply:
Yeah, I definitely agree. Thanks for the suggestion!